frontend/src/create-document/create-document.js

```diff
@@ -58,20 +53,10 @@ module.exports = app => app.component('create-document', {
     }
     this.documentData += '}';
     
-    this.editor = new EditorView({
-      state: EditorState.create({
-        doc: this.documentData,
-        extensions: [
-          basicSetup,
-          javascript()
-        ]
-      }),
-      parent: this.$refs.codeEditor
+    this.editor = CodeMirror.fromTextArea(this.$refs.codeEditor, {
```

Comment:
Initial value doesn't seem to show up? In the following create modal, the value of `this.documentData` should show up

![image](https://github.com/mongoosejs/studio/assets/1620265/b762620f-99ad-41cb-8071-1828e9d98a1c)